src/graph.rs

```diff
@@ -2388,6 +2388,7 @@ pub(crate) fn parse_js_module_from_module_info(
             maybe_resolver,
             maybe_npm_resolver,
           );
+          dep.maybe_deno_types_specifier = Some(specifier_text);
```

Comment:
Maybe we should rename this `maybe_types_specifier`? The problem here is that this was getting lost with `Dependency::with_new_resolver`

Reply:
Oh I see - yes, I think that makes sense.